source/shared/cpp/ObjectModel/CollectionTypeElement.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "pch.h"
+#include "BaseCardElement.h"
+
+namespace AdaptiveSharedNamespace
+{
+    class CollectionTypeElement : public BaseCardElement
+    {
+    public:
+        CollectionTypeElement(const CollectionTypeElement&) = default;
+        CollectionTypeElement(CollectionTypeElement&&) = default;
+        CollectionTypeElement& operator=(const CollectionTypeElement&) = default;
+        CollectionTypeElement& operator=(CollectionTypeElement&&) = default;
+        ~CollectionTypeElement() = default;
+        CollectionTypeElement(CardElementType type, ContainerStyle style = ContainerStyle::None,
+            VerticalContentAlignment alignment = VerticalContentAlignment::Top);
+
+        ContainerStyle GetStyle() const;
+        void SetStyle(const ContainerStyle value);
+
+        VerticalContentAlignment GetVerticalContentAlignment() const;
+        void SetVerticalContentAlignment(const VerticalContentAlignment value);
+
+        bool GetPadding() const;
+        void SetPadding(const bool value);
+
+        bool GetBleed() const;
+        void SetBleed(const bool value);
+        // tells if current element can bleed 
+        // when GetCanBleed() return false and GetBleed() returns true 
+        // the user must also check if HostConfig has
```

Comment:
>user [](start = 15, length = 4)

should this be `renderer`?

Reply:
i meant to be implementer of this feature or the renderer.
If it makes more clear, i will change it.

---
In reply to: [259076191](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r259076191) [](ancestors = 259076191)